src/backend/base/langflow/main.py

```diff
@@ -137,11 +137,8 @@ async def flatten_query_string_lists(request: Request, call_next):
 
         return await call_next(request)
 
-    @app.get("/health")
```

Comment:
I think we should keep this for now until we update the frontend.

Reply:
@ogabrielluiz `/health` is retained. It's just moved here in another file.
https://github.com/langflow-ai/langflow/pull/2510/files#diff-5f75901a0969078305ed5f9087a232612ba48e19ef19a2af437c93268e8cedacR36

Reply:
Sounds good. Sorry